backend/API/swagger_API.yaml

```diff
@@ -1,39 +1,34 @@
 swagger: "2.0"
 info:
-  description: "This is the API for AI Feedback Coach - AD440 Winter 2022"
+  description: "This is the API for AI feedback Coach - AD440 Winter 2022"
   version: "1.0"
-  title: "AI Feedback Coach API"
+  title: "AI feedback Coach API"
   termsOfService: "http://swagger.io/terms/"
   contact:
     email: "ericknigge@gmail.com"
   license:
     name: "Apache 2.0"
     url: "http://www.apache.org/licenses/LICENSE-2.0.html"
-host: "api.tasks.testsite01.com"
+host: "unknown"
 basePath: "/"
 paths:
 
-  /Student/{studentID}/:
+  /feedback:
     post:
       tags:
-      - "Student"
-      summary: "Create student and track feedback"
-      description: "Create student record and add teacher feedback for student"
-      operationId: "addStudentFeedback"
+      - "feedback"
+      summary: "Add feedback to Database"
+      description: "Add feedback to database"
+      operationId: "addfeedback"
       consumes:
       - "application/json"
       produces:
       - "integer"
       parameters:
-      - in: path
-        name: studentID
-        type: integer
-        required: true
-        description: "student ID number"
       - in: "body"
         name: "body"
-        description: "Student Feedback"
-        required: false
+        description: "Student feedback"
```

Comment:
Is it "Student" feedback or "AI" feedback as mentioned below?

Reply:
Correct student vs. AI references in file.